### PR TITLE
Improve Landing Page and Quickstart Guide

### PR DESCRIPTION
**Pull request by callumforrester**
Addresses some of the concerns raised in #279 

Changes:
* Add concept explanation and diagram to documentation landing page
* Add brief description and testing instructions to quickstart guide

### CODE REVIEW

**Comment by callumforrester** [comment]
Preview of landing page here: https://github.com/DiamondLightSource/blueapi/tree/welcome-page
Preview of docs changes here: https://diamondlightsource.github.io/blueapi/welcome-page/index.html

File: README.rst [hunk 1]
```diff
@@ -12,7 +12,28 @@ Documentation  https://DiamondLightSource.github.io/blueapi
 Releases       https://github.com/DiamondLightSource/blueapi/releases
 ============== ==============================================================
 
-TODO: Talk about automagic plan endpoints
+Wrap bluesky_ plans and devices inside a server and send commands/receive data.
```
Comment (DominicOram) [inline README.rst:15]: Nit: The tense here is a bit odd
```suggestion
This module wraps bluesky_ plans and devices inside a server and exposes endpoints to send commands/receive data.
```

**Review by DominicOram — CHANGES_REQUESTED** [review]
Great, work, thanks. Not strictly part of this PR I guess but the CLI commands on `Quickstart`->`Start Worker` do not work for my latest install of `blueapi`

File: README.rst (hunk 1 above)
Comment (callumforrester) [reply README.rst:15]: Agree, that sounds much better

**Comment by callumforrester** [comment]
@DominicOram The CLI commands work for me, could you confirm which version of blueapi you are using?
These commands should work for `>=0.3.0`

**Review by DominicOram — APPROVED** [review]
My bad, all good with latest `blueapi` version

**Pull request merged**